data-visualization/json/index08.js

```diff
@@ -0,0 +1,20 @@
+let myTeam = {
+  "Team": "Argentina",
+  "Sport": "Football",
+  "Year": 1986,
+  "World Cup Winner": true,
+  "Head Coach": {
+    "Name": "Carlos Bilardo",
+    "Matches": 7
+  },
+  "Players": [
+    {
+      "Name": "Sergio Batista",
+      "Position": "MF"
+    }
+  ]
+};
+
+/*
+Add two more key-value pairs to Sergio's object: `Number` set to a numeric value of `2`, and `isCaptain` set to a Boolean value of `false`.
```

Comment:
Add a sentence to describe the `isCaptain` notation. (like the lowercase of the word `is` and how it is commonly used to denote a boolean property)